web/viewer.css

```diff
@@ -1249,8 +1305,7 @@ html[dir="rtl"] .toolbarField[type="checkbox"] {
 .toolbarLabel {
   min-width: 16px;
   padding: 6px;
-  margin: 2px;
-  border: 1px solid rgba(0, 0, 0, 0);
```

Comment:
This change moves/shrinks the `#findResultsCount` label, please avoid that.

Reply:
The only way to remove the border in high contrast mode is by removing this line.
![Screenshot 2021-03-16 123114](https://user-images.githubusercontent.com/25604837/111278898-024cb380-8632-11eb-8862-97da1f36caee.png)

Reply:
Hmm, what about something like this then (the diff is against `master`, hopefully that's not a problem)?
```diff --git a/web/viewer.css b/web/viewer.css
index 442f57580..4a45149b8 100644
--- a/web/viewer.css
+++ b/web/viewer.css
@@ -660,7 +660,7 @@ html[dir="ltr"] .doorHangerRight:before {
   background-color: rgba(217, 217, 217, 1);
   color: rgba(82, 82, 82, 1);
   text-align: center;
-  padding: 3px 4px;
+  padding: 4px 5px;
   margin: 5px;
 }
 
@@ -1248,9 +1248,8 @@ html[dir="rtl"] .toolbarField[type="checkbox"] {
 
 .toolbarLabel {
   min-width: 16px;
-  padding: 6px;
+  padding: 7px;
   margin: 2px;
-  border: 1px solid rgba(0, 0, 0, 0);
   border-radius: 2px;
   color: var(--main-color);
   font-size: 12px;
@@ -1261,10 +1260,10 @@ html[dir="rtl"] .toolbarField[type="checkbox"] {
 }
 
 html[dir="ltr"] #numPages.toolbarLabel {
-  padding-left: 2px;
+  padding-left: 3px;
 }
 html[dir="rtl"] #numPages.toolbarLabel {
-  padding-right: 2px;
+  padding-right: 3px;
 }
 
 #thumbnailView {
```